build/Containerfile.controlplane

```diff
@@ -18,6 +18,8 @@ COPY controlplane/ controlplane/
 
 COPY dataplane/ dataplane/
 
+COPY tests/ tests/
+
```

Comment:
This should not be required in the container image right?

Reply:
Ok. There's probably other ways we can manage this (having a different Cargo workspace for builds) but this will be quicker for now, so let's keep it for now. :+1: